x/operator/keeper/slash.go

```diff
@@ -33,215 +33,98 @@ func GetSlashIDForDogfood(infraction stakingtypes.Infraction, infractionHeight i
 	return string(assetstype.GetJoinedStoreKey(hexutil.EncodeUint64(uint64(infraction)), hexutil.EncodeUint64(uint64(infractionHeight))))
```

Comment:
Please add the `chainID` or the hex address in this, since the same logic will be used to slash appchains as well.

Reply:
The key will include the chainID for the other appchains when storing the slash information.